backend/Services/DbServices/AuthServices.cs

```diff
@@ -0,0 +1,72 @@
+﻿using DataBase.Contexts;
+using Microsoft.EntityFrameworkCore;
+
+using Services.Dtos;
+using Services.Exceptions;
+using Services.Interfaces;
+
+namespace Services.DbServices
+{
+    internal class AuthServices : IAuthServices
+    {
+        private readonly DbContextMain _dbContext;
+        public AuthServices(DbContextMain context)
+        {
+            _dbContext = context;
+        }
+        public async Task Create(UserAuthModelDto model)
+        {
+            await _dbContext.AuthorisedUsers.AddAsync(model.MapToEntity());
+            await _dbContext.SaveChangesAsync();
+        }
+        public async Task Update(UserAuthModelDto newModel)
+        {
+            var entity = await _dbContext.AuthorisedUsers.SingleOrDefaultAsync(entity => entity.Id == newModel.Id);
+
+            if (entity is null)
+            {
+                throw new UserNotFoundException();
```

Comment:
Usually we don't use so common exceptions in a specific way. I mean you can create exception ObjectNotFoundException instead UserNotFoundException with constructor for name of 'not found' entity.